File: python progs/clock.cpp
/* write a clock computer program */

#include <iostream>
#include <ctime>
#include <string>

using namespace std;

int main(){
    // Get the current time
    time_t now = time(0);
    tm* ltm = localtime(&now);

    // Output the current time
    cout << "The current time is " << ltm->tm_hour << ":";
    cout << ltm->tm_min << ":" << ltm->tm_sec << endl;

    return 0;
}

File: python progs/clock.py
# ChatGPT response from:
# write a clock computer program

import time 

while True:
    currentTime = time.strftime("%H:%M:%S")
    print(currentTime)
    time.sleep(1)


"""
ian@hp:~/openai-quickstart-node/notes/python progs$ python clock.py
11:47:33
11:47:34
11:47:35

"""
